specification/databricks/resource-manager/Microsoft.Databricks/stable/2024-05-01/accessconnector.json

```diff
@@ -0,0 +1,438 @@
+        "tags": [
+          "AccessConnector"
+        ],
+        "operationId": "AccessConnectors_ListBySubscription",
+        "description": "Gets all the azure databricks accessConnectors within a subscription.",
+        "x-ms-examples": {
+          "Lists all the azure databricks accessConnectors within a subscription.": {
+            "$ref": "./examples/AccessConnectorsListBySubscriptionId.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns an array of azure databricks accessConnectors.",
+            "schema": {
+              "$ref": "#/definitions/AccessConnectorListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../preview/2022-04-01-preview/databricks.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AccessConnector": {
+      "type": "object",
+      "properties": {
+        "identity": {
+          "$ref": "../../../../../common-types/resource-management/v3/managedidentity.json#/definitions/ManagedServiceIdentity"
+        },
+        "systemData": {
+          "description": "The system metadata relating to this resource",
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/systemData"
+        },
+        "properties": {
+          "description": "Azure Databricks accessConnector properties",
+          "$ref": "#/definitions/AccessConnectorProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../preview/2022-04-01-preview/databricks.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "Information about azure databricks accessConnector."
+    },
+    "AccessConnectorProperties": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "description": "Provisioning status of the accessConnector.",
+          "readOnly": true,
+          "enum": [
+            "Deleted",
+            "Failed",
+            "Succeeded"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "referedBy": {
+          "description": "List of workspaces referring this accessConnector.",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "AccessConnectorUpdate": {
+      "description": "An update to an azure databricks accessConnector.",
```

Comment:
>azure [](http://example.com/codeflow?start=38&length=5)

Azure or Azure Databricks would be capitalized as a proper noun I think? 
(I am guessing Databricks is one of those proper noun 'brand name' things)

Reply:
Yes, you are correct, @curiekim0 can you update the description to make sure "Azure" and "Azure Databricks" has correct capitalization as applicable all the places?

Reply:
Updated all the capitalizations of all instances of "Azure", "Azure Databricks" and "Access Connector"